requirements.txt

```diff
@@ -10,5 +10,5 @@ nfselib.ginfes==0.2.0
 nfselib.issnet==0.2.0
 xmldiff==2.4
 lxml==4.6.2
--e git://github.com/erpbrasil/nfelib.git#egg=nfelib
+-e git://github.com/akretion/nfelib.git@master_gen_v4_00#egg=nfelib
```

Comment:
It would be important to leave the official lib or switch to akretion/nfelib (and merge it into the erpbrasil repo) because it can make installation difficult

Reply:
problem is KMEE is using the erpbrasil/nfelib fork in production, if we force the merge in their erpbrasil.edoc repo we may break their stuff in production. They didn't revised https://github.com/erpbrasil/erpbrasil.edoc/pull/30

I'm a bit pissed off to have spent time showing generateDS and akretion/nfelib to them and now get the project tied to a sub optimal nfelib forked version full of manual hacks again while the whole purpose of the code generation technology is to get rid of manual coding and manutention. But we can give them more time to make the switch.

Also I criticized several times that erpbrasil.edoc has hard dependencies in the binding libs. So basically we started depending on erpbrasil.edoc for some small nfse modules while the nfe part was still not clean and now we have exactly the situation I was warning about...

At the same time, it seems KMEE is using its own 12 staging branch instead the OCA 12 branch, so I would say we could still merge this one while leaving them more tome to make the switch in the erpbrasil.edoc repo.

Reply:
one more consideration in favor of the merge: in fact, at this stage using akretion/nfelib will only affect if the tests of this module pass or not (tests will only work with akretion/nfelib) but if KMEE wants to use erpbrasil/nfelib in their own requirements.txt, the code of the module will still be working exactly the same, because in fact it is quite independant from its bindings lib. So I would suggest a quick merge as this will not even impact people still using erpbrasil/nfelib in their production combo. cc @renatonlima @marcelsavegnago

Reply:
I agree with you about manual hacks in nfelib, may we can to use akretion/nfelib master branch because until these fixes are fixed in erpbrasil we can use akretion/nfelib